deploy/crds/mayadata.io_dmaasbackups.yaml

```diff
@@ -44,6 +44,10 @@ spec:
                   description: CronTime is cron expression defining when to run full
                     backup
                   type: string
+                disableSuccessfulBackupRetain:
```

Comment:
how about changing to `disableSuccessfulBackupCheckForRetention`?

Reply:
ok